src/main/java/com/mathworks/ci/BuildArtifactAction.java

```diff
@@ -220,6 +220,28 @@ private void iterateAllTaskAttributes(Entry pair, BuildArtifactData data) {
             case "skipped":
                 data.setTaskSkipped((Boolean) pair.getValue());
                 break;
+            case "skipReason":
+                String skipReasonKey = pair.getValue().toString();
+                String skipReason;
+                switch(skipReasonKey){
+                    case "UpToDate":
+                        skipReason = "Up-To-Date";
+                        break;
+                    case "UserSpecified":
+                        skipReason = "User Specified";
```

Comment:
Why do we have have both UserSpecified and UserRequested? Shouldn't we have UserRequested only?

Reply:
Yeah, Sam is correct. We need UserSpecified for releases prior to 25a. Although, Mark suggested we update the text to show "user requested" for both UserSpecified and UserRequested.

Reply:
Yes, I think "user requested" is clear enough and can cover all MATLAB releases.